client/layout/index.jsx

```diff
@@ -168,19 +169,28 @@ Layout = React.createClass( {
 				<TranslatorLauncher
 					isEnabled={ translator.isEnabled() }
 					isActive={ translator.isActivated() }/>
+				{ config.isEnabled( 'design-layout' ) && this.props.section.group === 'sites' &&
+					<DesignPreview
+						className="layout__design"
+						showPreview={ this.props.focus.getCurrent() === 'design' }
+					/>
+				}
 			</div>
 		);
 	}
 } );
 
-export default connect(
-	( state ) => {
-		const { isLoading, section } = state.ui;
-		return {
-			isLoading,
-			isSupportUser: state.support.isSupportUser,
-			section,
-			isOffline: isOffline( state )
-		};
-	}
-)( Layout );
+function mapStateToProps( state ) {
+	const { isLoading, section, hasSidebar, isFullScreen, chunkName } = state.ui;
```

Comment:
Oops, thanks! Those must have been left over from my old branch. I think I'll just revert this chunk anyway, since I'm not actually using it in this PR.